src/App.js

```diff
@@ -67,8 +71,16 @@ import { useDimensions } from '/libs/dimensions'
 import { configureFileLogger } from '/app/domain/logger/fileLogger'
 import { useColorScheme } from '/app/theme/colorScheme'
 
+if (__DEV__) {
+  require('react-native-performance-flipper-reporter').setupDefaultFlipperReporter()
+}
+
+configurePerformances()
 configureFileLogger()
 
+const markStartName = rnperformance.mark(markStartName)
```

Comment:
rnperformance.mark arg should be a string ?

Reply:
Nice catch, it was a refactoring error. Fixed [here](https://github.com/cozy/cozy-flagship-app/compare/9f81a021acfaa8a142ca2126adf3d84f0cea775e..e76e118d3a4da722258e471225978592b0390bba)